README.md

```diff
@@ -36,6 +36,7 @@ that make up the Bento platform.
 * [Development](./docs/development.md)
 * [Troubleshooting guide](./docs/troubleshooting.md)
 * [Deployment](./docs/deployment.md)
+* [Monitoring](./docs/monitoring.md)
```

Comment:
unrelated to this PR, but after this line (or in the section below perhaps, or in a new section called `Customization`) can you add a link to the public discovery config docs?

Reply:
Missed this, good point.